src/tests/fake/subtractions.ts

```diff
@@ -77,6 +77,14 @@ export function createFakeSubtractionMinimal(overrides?: CreateFakeSubtractionMi
     return assign(defaultSubtractionMinimal, overrides);
 }
 
+export function createFakeSubtractionShortlist() {
```

Comment:
Hmm naming seems off here. A subtraction shortList should be a list of these reduced subtraction objects